.evergreen.yml

```diff
@@ -41,6 +41,40 @@ functions:
           GOROOT: ${go_root}
           <<: *go_options
         command: make test
+  "atlas e2e":
+    - command: subprocess.exec
+      type: test
+      params:
+        add_to_path:
+          - ${go_bin}
+        working_dir: src/github.com/mongodb/mongocli
+        env:
+          WORKDIR: ${workdir}
+          GOROOT: ${go_root}
+          <<: *go_options
+          MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
+          MCLI_ORG_ID: ${mcli_org_id}
+          MCLI_PRIVATE_API_KEY: ${mcli_private_api_key}
+          MCLI_PROJECT_ID: ${mcli_project_id}
+          MCLI_PUBLIC_API_KEY: ${mcli_public_api_key}
+        command: make e2e-test-atlas
+  "iam e2e":
+    - command: subprocess.exec
+      type: test
+      params:
+        add_to_path:
+          - ${go_bin}
+        working_dir: src/github.com/mongodb/mongocli
+        env:
+          WORKDIR: ${workdir}
+          GOROOT: ${go_root}
+          <<: *go_options
+          MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
+          MCLI_ORG_ID: ${mcli_org_id}
```

Comment:
non blocking and can be done later, we may need to reconsider this name `mcli_org_id` as we are going to add cloud manager later and the id may be different, so I would do something like 
` MCLI_ORG_ID: ${atlas_org_id}` and later add a ` MCLI_ORG_ID: ${cloud_manager_org_id}` for cloud manager

We can address this in another PR but wanted to let you know

Reply:
I am going to do this change in the next PR